frontend/src/components/ScoreBadge.tsx

```diff
@@ -8,7 +8,7 @@ export function calculateScorePercent(score: number, maxScore: number): number {
         return 0;
     }
 
-    if (maxScore === 0) {
+    if (!maxScore) {
```

Comment:
Is this a biome thing? Checking the maxScore is zero seems much more readable than checking if not maxScore

Reply:
It’s intended to handle null/undefined instead of just 0. This was fixing some runtime errors I hit while messing with stuff, iirc

Reply:
But I should clarify it’s not necessary for this set of changes, it just snuck in from local testing I was doing. Happy to revert if desired

Reply:
ohh okay. that's fine, I didn't realize the value was ever not numeral